src/Logging.jl

```diff
@@ -86,25 +127,47 @@ show(io::IO, logger::Logger) = print(io, "Logger(", join([logger.name,
                                                           logger.output,
                                                           logger.parent.name], ","), ")")
 
-const _root = Logger("root", WARNING, STDERR)
-Logger(name::AbstractString;args...) = configure(Logger(name, WARNING, [STDERR], _root); args...)
+const _root = Logger("root", WARNING, LogIO(STDERR))
+Logger(name::AbstractString; kwargs...) = configure(Logger(name, WARNING, [LogIO(STDERR)], _root); kwargs...)
 Logger() = Logger("logger")
 
 write_log(syslog::SysLog, color::Symbol, msg::AbstractString) = send(syslog.socket, syslog.ip, syslog.port, length(msg) > syslog.maxlength ? msg[1:syslog.maxlength] : msg)
-write_log{T<:IO}(output::T, color::Symbol, msg::AbstractString) = (print(output, msg); flush(output))
+write_log(output::IO, color::Symbol, msg::AbstractString) = (print(output, msg); flush(output))
```

Comment:
Not sure we want to do this. The benefit of keeping `IO` parametrized (as I have only recently learned) is that the compiler can optimize it better. If it is parametrized, it can create a new function for each value that T can hold. I'm not sure why it can't figure out how to do that without parametrization.